app/scripts/modules/ecs/src/serverGroup/details/events/serverGroupEventsReader.service.ts

```diff
@@ -0,0 +1,25 @@
+import { IPromise } from 'angular';
+
+import { $log } from 'ngimport';
+
+import { API } from '../../../../../core/src/api/ApiService';
```

Comment:
Import from `@spinnaker/core` here and below.

Reply:
👍